ruma-events/src/room/encrypted.rs

```diff
@@ -27,6 +26,22 @@ pub enum EncryptedEventContent {
     MegolmV1AesSha2(MegolmV1AesSha2Content),
 }
 
+/// The content payload for `EncryptedEvent`.
+#[derive(Clone, Debug, Deserialize, Serialize, MessageEventContent)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+#[ruma_event(type = "m.room.encrypted")]
+#[serde(tag = "algorithm")]
```

Comment:
This attribute needs to be removed, otherwise we'll serialize two algorithm fields. It's unfortunate that `serde_json::to_value` doesn't fail serialization in this case.